lib/stack.dart

```diff
@@ -24,27 +22,13 @@ class Stack<T> {
   }
 
   /// get the top of the stack without deleting it.
-  T top() {
-    return _list.last;
-  }
+  T top() => _list.last;
 
   /// get the size of the stack.
-  int size() {
-    return _list.length;
-  }
-
-  /// get the length of the stack.
-  int get length => size();
```

Comment:
@DebkanchanSamadder Why did you delete `length` here?

Reply:
As far as I know implicitly a stack has size not length. Moreover length and size do the same thing so it creates ambiguity.